assessments/projects/jaeger/self-assessment.md

```diff
@@ -0,0 +1,302 @@
+# Jaeger Self-assessment
+
+
+## Table of contents
```

Comment:
```suggestion
# Jaeger Self-assessment

This assessment was created by community members as part of the [Security Pals](https://github.com/cncf/tag-security/issues/1102) process and is currently pending changes from the maintainer team.

## Table of contents
```
@ragashreeshekar

Reply:
I am not sure why this is necessary, since that issue isn't related to the pending issues in this particular issue.

Reply:
We are providing this note for all the Self assessments completed through the TAG Security Pals initiative.